assets/section-footer.css

```diff
@@ -160,46 +155,94 @@
   }
 }
 
+.footer__follow-on-shop:empty,
 .footer__list-social:empty,
 .footer-block--newsletter:empty {
   display: none;
 }
 
+.footer__follow-on-shop {
+  display: flex;
+  text-align: center;
+}
+
 .footer__list-social.list-social:only-child {
   justify-content: center;
 }
 
 .footer-block__newsletter {
   text-align: center;
+  flex-grow: 1;
 }
 
 .newsletter-form__field-wrapper {
   max-width: 36rem;
 }
 
 @media screen and (min-width: 750px) {
+  /* Pushes other components to the right of the flexbox */
   .footer-block__newsletter:not(:only-child) {
     text-align: left;
+    margin-right: auto;
   }
 
   .footer-block__newsletter:not(:only-child) .footer__newsletter {
     justify-content: flex-start;
     margin: 0;
   }
 
-  .footer-block__newsletter:not(:only-child) .newsletter-form__message--success {
+  .footer-block__newsletter:not(:only-child)
+    .newsletter-form__message--success {
     left: auto;
   }
-}
 
-.footer-block__newsletter + .footer__list-social {
-  margin-top: 3rem;
+  .footer__follow-on-shop {
+    margin-bottom: 0.4rem;
+  }
+  /* Follow on shop is the first button but it has siblings*/
+  .footer__follow-on-shop:first-child:not(:last-child) {
+    justify-content: flex-start;
+    margin-right: auto;
+    text-align: left;
```

Comment:
Curious to see in which scenarios the text align applies 🤔 I feel like it doesn't much impact in what I tested.

Reply:
Same for the justify-content. It seems to be working with the `margin-right: auto;` fine and maybe not needing text align and justify-content 🤷 
But I might be missing something 😅